src/igvfd/schemas/in_vitro_system.json

```diff
@@ -590,6 +590,19 @@
                 "appscript": "DMEM with serum",
                 "igvf_utils": "DMEM with serum"
             }
+        },
+        "biosample_qualifier": {
+            "title": "Biosample qualifier",
+            "description": "Desription of cell state",
+            "type": "string",
+            "enum": [
+                "exhausted",
+                "stimulated"
```

Comment:
Another question (maybe for lab): do we need "stimulated" here or is it something that could be captured using Treatment objects?